inst/plumber/17-arguments/plumber.R

```diff
@@ -0,0 +1,126 @@
+#* Works with query string aruguments
+#* @serializer print
+#* @post /named/<a>/<b>
+function(a, b) {
+  list(a = a, b = b)
+}
+
+
+
+#* Can have conflicting path, query, and post body arugments
+#* @serializer print
+#* @post /args/<a>/<b>
+function(req, res) {
+  list(
+    all = req$args,
+    query = req$argsQuery,
+    path = req$argsPath,
+    postBody = req$argsPostBody
+  )
+}
+
+if (FALSE) {
+  # Example of failure to call a function with multiple named arguments
+  do.call(function(a, ...) {list(a, ...)}, list(a = 1, b = 2, a = 3))
+  #> Error in (function (a, ...)  :
+  #> formal argument "a" matched by multiple actual arguments
+}
```

Comment:
I think this might end up being more confusing than it is helpful for most users...if you want to keep it maybe let's put it at the end of the file as an afterthought

Reply:
I'm ok with taking it out